index.html

```diff
@@ -5,8 +5,59 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link rel="stylesheet" href="styles/index.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.1.1/css/fontawesome.min.css">
+
 </head>
 <body>
-    
+    <main>
+        <h1>Weather Report</h1>
+            <h3 class="top-city-title">For the lovely city of: <h3 id="city-title-name"></h3></h3>
```

Comment:
This probably shouldn't have been a `h3` (and especially the nested one shouldn't have been) since it doesn't actually title a portion of the page and you skipped right from `h1` to `h3`.

I'd recommend this as a `p` with a `span` in it:

```suggestion
            <p class="top-city-title">For the lovely city of: <span id="city-title-name"></span></p>
```

Remember, if something isn't the right size you can always resize it in CSS!